hodu-core/src/languages.rs

```diff
@@ -1,45 +1,27 @@
-use serde::{Deserialize, Serialize};
+use crate::sandbox::Sandbox;
 
-mod c;
-mod cpp;
-mod java;
-mod javascript;
-mod python;
+pub mod c;
+pub mod cpp;
+pub mod java;
+pub mod javascript;
+pub mod python;
 
-#[derive(Deserialize)]
-pub enum Language {
-    C,
-    CPP,
-    JAVA,
-    PYTHON,
-    JAVASCRIPT,
+pub trait LanguageExecutor {
+    async fn run<S: Sandbox>(&self, code: &str, sandbox: &S) -> ExecutionResult;
```

Comment:
P3) 제네릭을 쓰신 이유가 있나요?
이렇게 바꿔줬으면 하는 건 아니고 어떻게 하든 상관은 없는데 순수하게 궁금해서요 ㅎㅎ
```suggestion
    async fn run(&self, code: &str, sandbox: &impl Sandbox) -> ExecutionResult;
```

Reply:
아 이렇게하는거였군 [remove generic](https://github.com/wafflestudio/hodu/pull/25/commits/c67a4e9c73c2181f039c0a8d3cacf04a751dd414) 변경했슴다 감사감사